src/View/Sidebar.vala

```diff
@@ -580,7 +580,7 @@ public class Marlin.Sidebar : Marlin.AbstractSidebar {
         store.clear ();
 
         iter = add_category (PlaceType.BOOKMARKS_CATEGORY,
-                             _("Personal"),
+                             _("Favorites"),
                              _("Your common places and bookmarks"));
```

Comment:
Maybe:

```suggestion
                             _("Your common places and saved files and folders"));
```

@cassidyjames ?

Reply:
Oh, that one passed by me :facepalm: 

Your suggestion makes sense. But I'll wait for Cass's OK.

Reply:
The two "and"s feels stilted. Don't we also typically avoid second-person? Uuuuuuuhhhhh… how about just

```suggestion
                             _("Saved folders and files"));
```

Reply:
But the "common places" part I think is important to mention, it's those "Documents, Recents, etc" options. :)

Reply:
There's also "Recent" and "Home" which are not optional. I guess we don't _need_ to include that in the tooltip, but if we could figure something out about it that would be nice :)

Reply:
This is getting really hairy, but I think we've used "plus" before for clarity instead of two "and"s. But we're also using "place" and "folders and files," so maybe this could be simplified.

`Common places plus saved folders and files` or `Common and saved places` could work depending on how granular we want to get.

Reply:
>`Common places plus saved folders and files`

I like this approach. :D

Reply:
Yeah I'm +1 to calling out "folders and files" specifically so users know they have save files as well

Reply:
should it be "files and folders" though because alphabet?

Reply:
imo, folders and files retains some hierarchy that mimics the sort order Files does it.

Reply:
Going to go with `Common places plus saved folders and files` here since it seems like it's the preferred choice. :)